### DISCUSSION TITLE
contact

### DISCUSSION THREAD

**Discussion by giscus** [narwhrl/narwhrl.github.io#7] [Giscus]
# contact

https://video.ncc17039.top/contact

**Comment by Bashir-1** [comment]
# 关于评论系统
- 本评论系统使用Gisqus；
- 需要使用GitHub登录；
- 您上传图片可以使用外部图床，推荐使用：https://imgse.com/ 上传完成后，它会自动给你一个Markdown代码，直接复制粘贴到此处就行。
# 评论区用来做什么
**此评论区可以用来反馈问题、交流、讨论、建议/意见。**
